setup.py

```diff
@@ -0,0 +1,26 @@
+from setuptools import setup
+
+setup(
+    name="noobaa-sa-ci",
+    version="0.1",
+    packages=[""],
+    url="",
+    license="MIT",
+    author="Noobaa SA QE",
+    author_email="ocs-ci@redhat.com",
+    description=(
+        "Noobaa Standalone(SA) infra is used to deploy, upgrade and destroy NooBaa standalone on "
+        "RHEL/CentOS stream platforms"
+    ),
+    install_requires=[
+        "mergedeep",
+        "pytest",
```

Comment:
I think we discussed we will avoid the pytest in this repo.

Just will have entry point for `noobaa-sa-deploy`, ` noobaa-sa-destroy` which will just run functionality and we will not have any test here. The test of verification of the deployment will be later on in `noobaa-sa-ci` repo with the rest of the tests.

Reply:
done